src/sentry/api/endpoints/dashboards.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import
+
+
+from django.db import IntegrityError, transaction
+from sentry.api.base import DocSection
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.bases.dashboard import DashboardSerializer
+from sentry.api.serializers import serialize
+from sentry.models import Dashboard
+from rest_framework.response import Response
+
+
+class OrganizationDashboardsEndpoint(OrganizationEndpoint):
+    doc_section = DocSection.ORGANIZATIONS
+
+    def get(self, request, organization):
+        """
+        Retrieve an Organizations Dashboards
+        `````````````````````````````````````
+
+        Retrieve a list of dashboards that are associated with the given organization.
+
+        :pparam string organization_slug: the slug of the organization the
+                                          dashboards belongs to.
+        :qparam string query: the title of the dashboard being searched for.
+        :auth: required
+        """
+        dashboards = Dashboard.objects.filter(
+            organization_id=organization.id
+        )
+        query = request.GET.get('query')
+        if query:
+            dashboards = dashboards.objects.filter(
+                title=query,
+            )
+
+        context = serialize(list(dashboards), request.user)
+        return Response(context)
+
+    def post(self, request, organization):
```

Comment:
Shouldn't this be in a different endpoint? This should give us a list of dashboards and we should create a dashboard via the dashboard details endpoint (singular)

Reply:
So @billyvg initially I thought that too, but from what I've seen of the sentry api there are a lot of examples where that is not how we seem to be doing it. I looked at examples where the endpoint had both a <endpoint_name>s.py and a <endpoint_name>_details.py with both a post and put method. The examples such as: 

- https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/api_applications.py
- https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/group_notes.py
- https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_release_files.py

There are more examples, but I was following those. Could you give me a better sense of why our case is different from those?